src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -200,9 +200,11 @@ private void CheckFileSystem()
                 // Check for marker file indicating a zip package failure, and if found stop the application.
                 // We never want to run with an incorrect file system.
                 string path = Path.Combine(_applicationHostOptions.CurrentValue.ScriptPath, ScriptConstants.RunFromPackageFailedFileName);
+
                 if (File.Exists(path))
                 {
-                    _logger.LogError($"Shutting down host due to presence of {path}");
+                    string errorMessage = File.ReadAllText(path);
+                    _logger.LogError($"Shutting down host due to presence of {path}. Error details: {errorMessage}");
                     _applicationLifetime.StopApplication();
                 }
```

Comment:
If ZipDeploymentAppSettingsExist is truthy and this file does not exist, should we log a warning?

Reply:
yes i think we should

Reply:
From my understanding, `ZipDeploymentAppSettingsExist = true` and this file not existing is the expected behavior/happy path?

Reply:
And we would only stop the host if `FAILED TO INITIALIZE RUN FROM PACKAGE.txt` exists.